### ISSUE TITLE
.google.bigtable.v2.RowRange#startkeyClosed is not a field

### ISSUE THREAD

**Issue by arbesfeld** [GoogleCloudPlatform/google-cloud-node#1554]
Hi, I am getting this error when trying to `getRows` on `0.2.0`. I am using `0.2.0` to get `decode: false` behavior.:

```
this.table.getRows({
  start: 'start',
  end: `end`,
});
```

```
Error: .google.bigtable.v2.RowRange#startkeyClosed is not a field: undefined
    at Error (native)
    at [object Object].MessagePrototype.set (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2440:35)
    at [object Object].MessagePrototype.set (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2434:38)
    at [object Object].Message (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2363:34)
    at [object Object].ProtoBuf.Reflect.ElementPrototype.verifyValue (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:1877:28)
    at [object Object].ProtoBuf.Reflect.FieldPrototype.verifyValue (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:3417:43)
    at [object Object].MessagePrototype.set (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2443:59)
    at [object Object].MessagePrototype.set (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2434:38)
    at [object Object].Message (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2363:34)
    at [object Object].ProtoBuf.Reflect.ElementPrototype.verifyValue (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:1877:28)
    at [object Object].ProtoBuf.Reflect.FieldPrototype.verifyValue (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:3436:33)
    at [object Object].MessagePrototype.set (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2443:59)
    at [object Object].MessagePrototype.set (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2434:38)
    at [object Object].Message (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/protobufjs/dist/ProtoBuf.js:2363:34)
    at serialize (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/@google-cloud/common/node_modules/grpc/src/node/src/common.js:87:23)
    at Client.makeServerStreamRequest [as readRows] (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/@google-cloud/common/node_modules/grpc/src/node/src/client.js:550:19)
    at Object.retryOpts.request (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/@google-cloud/common/src/grpc-service.js:333:39)
    at makeRequest (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/retry-request/index.js:80:28)
    at retryRequest (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/retry-request/index.js:58:3)
    at Bigtable.GrpcService.requestStream (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/@google-cloud/common/src/grpc-service.js:348:10)
    at Instance.GrpcServiceObject.requestStream (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/@google-cloud/common/src/grpc-service-object.js:111:22)
    at Table.GrpcServiceObject.requestStream (/Users/mata/AppHub/logrocket/packages/@apphub:logrocket-server-storage-bigtable/node_modules/@google-cloud/common/src/grpc-service-object.js:111:22)
```

**Comment by stephenplusplus** [comment]
Sorry about that :( @callmehiphop I'm guessing this should be startKeyClosed?

**Comment by callmehiphop** [comment]
Yep! Going to fix ASAP

**Comment by callmehiphop** [comment]
Thanks for reporting this @arbesfeld, I just released a patch so if you update to `0.2.1` this should be resolved.

**Comment by arbesfeld** [comment]
@callmehiphop thanks so much for the quick fix!
